src/lib/dialog/dialog.md

```diff
@@ -29,6 +29,37 @@ Components created via `MdDialog` can _inject_ `MdDialogRef` and use it to close
 in which they are contained. When closing, an optional result value can be provided. This result
 value is forwarded as the result of the `afterClosed` promise. 
 
+### Sharing Data with the Dialog component.
+Depending on what you are doing you might want to share data with your dialog component. The dialog component has a data option that you can include in order to pass data from the open to the dialog. 
```

Comment:
You should add backticks(\`) around "data" in "The dialog component has a data option".